docs/DeveloperGuide.md

```diff
@@ -451,6 +457,193 @@ Given below are instructions to test the app manually.
+Expected: An error message along with a formatting guideline message will be printed in the command line interface..  
+
+#### 2.7 Finding a Task using a keyword
+
+2.7.1. Finding all tasks that contains a specified keyword.  
+i. Prerequistes: User must be in the Timetable mode. Enter `mode /t` command to enter Timetable mode. List all tasks in the tasklist using the `list` command. There must be existing tasks in the list.  
+ii. Test case: `Find Project`  
+Expected: All tasks with tags that contains the word 'Project' will be printed. If the tasks do not have tags, it will list the tasks with titles that contain the keyword 'Project'.  
+iii. Test case: `Find`  
+Expected: An error message along with a formatting guideline message will be printed in the command line interface.  
+
+#### 2.8 Adding a Notebook/Section/Page
+
+2.8.1. Adding a respective notebook/section/page in their respective list.  
+i. Prerequistes: User must be in the Notebook mode. Enter `mode /n` command to enter Notebook mode.    
+
+2.8.1.1. Adding a Notebook.  
+i. Prerequistes: User must not be in a selected notebook or section. User should type `mode /n` before executing this test case.   
+ii. Test case: `add /nCS2113T`  
+Expected: Notebook 'CS2113T' is added to the notebookShelf with a success message printed.  
+
+2.8.1.2. Adding a Section.  
+i. Prerequistes: User must select a notebook. Enter `select /s[NOTEBOOK]` to enter a specified notebook.    
+ii. Test case: `add /nChapter 1`  
+Expected: Section 'Chapter 1' is added to the specified notebook with a success message printed.  
+
+2.8.1.3. Adding a Page.  
+i. Prerequistes: User must select a Section. Enter `select /s[SECTION]` to enter a specified section.    
+ii. Test case: `add /p Page 1 ; a test page`  
+Expected: Page 'Page 1' along with content 'a test page' is added to the specified section with a success message printed.  
+
+#### 2.9 Selecting a Notebook/Section/Page
+
+2.9.1. Selecting a respective notebook/section/page in their respective list.  
+i. Prerequistes: User must be in the Notebook mode. Enter `mode /n` command to enter Notebook mode.  
+
+2.9.1.1. Selecting a Notebook.  
+i. Prerequistes: User must not be in a section. User should type `mode /n` before executing this test case.   
+ii. Test case: `select /nCS2113T`  
+Expected: The user will enter the selected notebook 'CS2113T' with a success message printed. 
+
+2.9.1.2. Selecting a Section.  
+i. Prerequistes: User must be in a selected notebook. Enter `select /s[NOTEBOOK]` to enter a specified notebook.   
+ii. Test case: `select /sChapter 1`  
+Expected: The user will enter the selected Section 'Chapter 1' with a success message printed. 
+
+2.9.1.3. Selecting a Page.  
+i. Prerequistes: User must be in a selected section. Enter `select /s[SECTION]` to enter a specified section.   
+ii. Test case: `select /sPage 1`  
+Expected: The user will enter the selected Page 'Page 1' with a success message printed. 
```

Comment:
not sure if you wanna add `select /nCS2113T /sChapter1 /p1` into this section because you can actually do that in our app...

Reply:
I can include that in for the test case